### ISSUE TITLE
Issue with tutorials/setting-up-lamp/index.md

### ISSUE THREAD

**Issue by ftntming** [GoogleCloudPlatform/community#1486]
Not working any more

**Comment by ToddKopriva** [comment]
@ftntming , can you give details of exactly what went wrong at what point in the document?

**Comment by ftntming** [comment]
> @ftntming , can you give details of exactly what went wrong at what point in the document?

I just tried and found the following about the tutorial

1. Does not cover Debian 10
2. Ubuntu 18.04 has problem installing Mysql 8 by apt (need to refer to another article to install it)
3. Ubuntu 20.04 had problem in "sudo mysql_secure_installation", in this particular step, even if I disabled the password security check plugin, and set a simple password 111111, it seems worked but actually it's not, the root password is still empty (took me several rounds to figure that out)...

I can't believe I spent 2 hours on trying to install LAMP on a GCP vm..
I never had such problems with AWS
